hab_gui/widgets/alias_button.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, cfg, alias_name, parent=None):
 
         self.alias_dict = self.cfg.aliases
         qsize_policy = QtWidgets.QSizePolicy
-        size_policy = qsize_policy(qsize_policy.Minimum, qsize_policy.Preferred)
+        policy = qsize_policy.Policy
+        size_policy = qsize_policy(policy.Minimum, policy.Preferred)
```

Comment:
Lets change this to fully qualified enum names so it can be caught by the enum check that is just looking at the text of the file.

```py
size_policy = qsize_policy(
    QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Preferred
)
```

Once that is done this is good to merge.